algorithms/sum_rec/sum.c

```diff
@@ -0,0 +1,15 @@
+#include<stdio.h>
+int findSum(int n)
+{
+  if(n<=1)
```

Comment:
This will make it return `n` for all negative numbers. For negative numbers, it should print an error message saying its not a natural number.